ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
 <template>
   <v-container
     id="admin-setting-view"
-    fluid
     tag="section"
   >
     <v-row justify="center">
-      <v-col
-        cols="8"
-        md="8"
-      >
-        <material-card
-          color="primary"
+      <v-col :cols="isMobile? 12 : 8">
+        <v-card
+          class="admin-settings-view__card"
+          :elevation="isMobile? 0 : 2"
         >
-          <v-img
-            class="rounded-circle mt-n12 d-inline-block"
-            :src="profilePicture"
-            width="128"
-          />
-          <v-form>
-            <v-container class="py-0">
-              <v-row>
-                <v-col
-                  cols="12"
-                  md="12"
-                >
-                  <v-text-field
-                    v-model="formData.email"
-                    disabled
-                    label="E-mail address"
-                  />
-                </v-col>
-
-                <v-col
-                  cols="12"
-                  md="12"
-                >
-                  <v-text-field
-                    v-model="formData.password"
-                    color="purple"
-                    label="Password"
-                  />
-                </v-col>
-
-
-                <v-col
-                  cols="12"
-                >
-                  <v-btn
-                    color="primary"
-                    min-width="150"
+          <v-row justify="center">
+            <v-col cols="4">
+              <v-img
+                class="rounded-circle d-inline-block mx-2 mb-5"
+                :src="require('@assets/svg/profile_picture.svg')"
+                width="128"
+              />
+            </v-col>
+          </v-row>
+          <ValidationObserver v-slot="{ handleSubmit }">
+            <v-form
+              v-model="isFormValid"
+              class="admin-settings-form"
+              @keyup.native.enter="handleSubmit(onFormSubmit)"
+              @submit.prevent="handleSubmit(onFormSubmit)"
+            >
+              <v-container class="py-0">
+                <v-row>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
                   >
-                    Update Profile
-                  </v-btn>
-                </v-col>
-              </v-row>
-            </v-container>
-          </v-form>
-        </material-card>
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="E-mail"
+                        rules="required|email"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.email"
+                        outlined
+                        required
+                        disabled
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                        class="my-0 py-0"
```

Comment:
With py-0, the horizontal padding is not nullified. I think its better to use pa-0 instead. 

![image](https://user-images.githubusercontent.com/12537724/172919289-c43060ae-9480-4bc4-89ac-21f4467b334c.png)